src/Serializer/ArraySerializerWithNull.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace League\Fractal\Serializer;
+
+class ArraySerializerWithNull extends ArraySerializer
```

Comment:
that serializer meant to demonstrate null():null, but i created file in wrong place.
i moved it up to test/stub directory
i don't wanted to make new ArraySerializer